### PR TITLE
Report whether implementation is nodejs

### PR DESCRIPTION
**Pull request by benmccann**
This is my first time using Akka, so make sure I didn't do anything stupid

### CODE REVIEW

File: src/main/scala/com/typesafe/jse/Engine.scala [hunk 1]
```diff
@@ -114,6 +114,8 @@ object Engine {
    */
   case class JsExecutionResult(exitValue: Int, output: ByteString, error: ByteString)
 
+  case class IsNode()
```
Comment (benmccann) [inline src/main/scala/com/typesafe/jse/Engine.scala]: done. thanks for taking a look!

**Comment by benmccann** [comment]
Thanks @wsargent. I've updated that. Anything else you think I should address?

**Comment by mkurz** [comment]
@benmccann You are probably working towards https://github.com/typesafehub/npm/issues/9? It would be nice to get a new release out before the Play 2.6 final release...

**Comment by benmccann** [comment]
Yes. Agreed on getting this out before Play 2.6. I need someone to cut a new release of this project before I can proceed though

**Pull request merged**